pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -233,7 +233,7 @@ spec:
         - name: ociStorage
           value: $(params.ociStorage)
         - name: sourceDataArtifact
-          value: "$(tasks.reduce-snapshot.results.sourceDataArtifact)"
+          value: "$(tasks.filter-already-released-advisory-images.results.sourceDataArtifact)"
```

Comment:
Isn't this broken? `apply-mapping` uses a `filter-already-released-advisory-images` result as an input param. `filter-already-released-advisory-images` has `runAfter: apply-mapping`